sap-labs-france/ev-dashboard
# Using DateTimeRangePicker with custom ranges instead of independent From, To and menu option to select custom ranges.

Test:
- Check if the datetime range picker works in Logs page as expected.
- Check if the date ranges shown in the datetime range picker selects the right start and end datetime values.
- Check if all the text involved is translated to respective language when locale is changed for all languages.
- Check if reset filters works as expected: reset startdate: 12:00 AM and endDate: now
- Check the same functionality for all the below pages
  - Sessions > History
  - Sessions > In Error
  - Invoices
  - Statistics (all tabs)

Current output (Updated image)
![image](https://user-images.githubusercontent.com/5779016/148852209-8a9c7f61-6332-4242-91ca-34d4a08ecbb0.png)
